README.md

```diff
@@ -1,16 +1,185 @@
-# Title
+## Programming as Collaboration: Nested Data Structures Introduction
 
-## Learning Goals
+Welcome to "Programming as Collaboration." We're going to learn to use the
+programming skills we know to work with _nested data structures_ (NDS).
 
--SWBAT 1
--SWBAT 2
+"Nested" means "complex" because we're talking about structures (`Array`s,
+`Hash`es) that contain _other_ structures (`Array`s, `Hash`es). A very simple
+"nested" data structure is an "`Array` of `Array`s:"
 
-## Introduction
+```ruby
+array_of_arrays = [ # containing Array (of Array)
+  [1, 3, 5], # "Nested" Array 1
+  [2, 4, 6]  # "Nested" Array 2
+]
+```
```

Comment:
Do we teach anything about multidimensional arrays? It might be useful to give a 1 sentence example of when a programmer might ever need to use one, and maybe model a code snippet after that. I used one in the Tic Tac Toe assignment before. I think it would have gaming applications. This isn't pressing, but I think it would be useful for students.